packages/experimental/PipsPager/src/PipsPager.tsx

```diff
@@ -0,0 +1,21 @@
+/** @jsx withSlots */
```

Comment:
@Saadnajmi I'm working on adding the PipsPager component by importing it from the react-native-xaml package. I'm hoping this can integrate with the FURN infrastructure well, but I'm not sure how much of the compose framework is necessary. Is there a way I can cut down on this method (for instead there aren't any tokens right now) while complying with FURN?

Reply:
@JasonVMo has been working on making new API's to let you create components without using compose, and may have a better answer. He's out till Wednesday though as an FYI. 

You can try looking at `stagedComponent`, which I think the `Icon` component uses.

Reply:
I'll look into those, thank you!